development-setup-wizard.py

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env python3
```

Comment:
Perhaps rename the file to use underscores instead of hyphens.  This would better match how Python scripts are normally named and better match the names of our other Python scripts.